src/localize.js

```diff
@@ -292,6 +293,7 @@ export const defaultTranslateOptions: InitializeOptions = {
   renderToStaticMarkup: false,
   renderInnerHtml: false,
   ignoreTranslateChildren: false,
+  defaultLanguage: '',
```

Comment:
Just wondering why you had to add this? Was it to get flow to stop complaining?

Reply:
well - i added it thinking you'd want the object to check for it since it should always be there (even if the default backup is used). i used an empty string to avoid any flow issues